scripts/west_commands/blobs.py

```diff
@@ -141,15 +144,26 @@ def verify_blob(self, blob):
                 URL:    {blob['url']}
                 Info:   {blob['description']}'''))
 
+            raise ChecksumError('Checksum mismatch')
```

Comment:
Up to you, but a `True` or `False` return value would be simpler than raising an exception? especially given that the function is called `verify_blob()`

Reply:
Good suggestion - fixed!